Codeinwp/otter-blocks
# feat: add more Woo conditions to Block Conditions module

Closes https://github.com/Codeinwp/otter-internals/issues/198.


### Summary

This PR adds:

- Product Category
- Product Tag
- Product Attribute

conditions to Block Conditions module.

### Screenshots 

----

### Test instructions


- You can import WooCommerce's sample data to get a good set of products.
- Confirm all the conditions work when the taxonomy is present/absent.
- There are no debugging errors.
- This is made for Product Page, so you must be using a FSE theme for being able to customize the Single Product Template.



---- 

### Checklist before the final review
